ibcalvariotv/transmissao/equipamentos-e-programas/controlador-ptz-vs-ptc-ip/index.md

```diff
@@ -4,4 +4,31 @@
 
 [Manual oficial](https://www.marshall-usa.com/pdf/Marshall_VS-PTC-IP_manualV3.pdf)
 
-TODO: Tradução das principais funcionalidades
+## Relação dos botões
+![Relação de botões](botoes_do_controle.png)
+
+Há várias funcionalidades no controlador de cameras, porém apenas algumas são utilizadas pelos operadores de PTZ. Estas são (Os números representam os números na imagem acima):
+
+5. Tela de LCD - Mostra informações como, qual camera está, se é possivel mexer, se preset foi salvo etc.
+8. Preset - Usado para salvar presets (Pressione e segure até que na tela apareça que o preset foi salvo).
+9. Controlador da velocidade do Pan (movimento horizontal) e Tilt (movimento vertical).
+10. Botão para chamar cenas - sempre a cena só será chamada quando ele for pressionado.
```

Comment:
Isso é para carregar o preset certo? Se for isso acho que vale usar a mesma linguagem usada no item 8, ou pelo menos fazer menção dela.
```suggestion
10. Carga de preset - Sempre a cena só será chamada quando ele for pressionado.
```